docs/index.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+  :description: ROCm SMI
+  :keywords: install, SMI, library, api, AMD, ROCm
+
+****************************************************
+ROCm System Management Interface (ROCm SMI) library
+****************************************************
+
+The ROCm System Management Interface library, or ROCm SMI library, is part of the ROCm software stack. It is a C library for Linux that provides a user space interface for applications to monitor and control GPU applications.
+
```

Comment:
Perhaps we should include a cross-reference to AMD SMI and explain the difference. 
And we should do the same for AMD SMI main page.